setup.cfg

```diff
@@ -40,6 +40,10 @@ package_dir =
 [options.packages.find]
 where = src
 
+[options.package_data]
+editables =
+    py.typed
```

Comment:
Excuse my ignorance but why do we need a `py.typed` file?

Reply:
A `py.typed` file signals to type checkers that the package is annotated - see [PEP 561](https://www.python.org/dev/peps/pep-0561/), "Packaging Type Information".

Reply:
Bluntly, that feels stupid to me. If the functions have type annotations, why isn't that sufficient? I'll have to read the PEP (thanks for the link!) to make sure I understand this. I'll follow up on this later.

Reply:
I believe the reasoning is that type checkers do not have to use type inference, which is expensive, and can assume that a `py.typed` package's exported types are correct - but don't quote me on that.